### PR TITLE
SSCS-10576 adds Correction Application to DocumentType and ScannedDocumentType

### PR DESCRIPTION
**Pull request by amoslv1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10576

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/DocumentType.java [hunk 1]
```diff
@@ -5,40 +5,41 @@
...
-    OTHER_EVIDENCE("Other evidence"),
-    OTHER_DOCUMENT("Other document"),
-    DWP_RESPONSE("DWP response", "FTA response"),
+    ADJOURNMENT_NOTICE("adjournmentNotice", "Adjournment Notice"),
+    APPELLANT_EVIDENCE("appellantEvidence", "Appellant evidence"),
+    AT38("at38", "AT38"),
+    AUDIO_DOCUMENT("audioDocument", "Audio document"),
+    AUDIO_VIDEO_EVIDENCE_DIRECTION_NOTICE("audioVideoEvidenceDirectionNotice", "Audio/Video evidence direction notice"),
+    CONFIDENTIALITY_REQUEST("confidentialityRequest", "Confidentiality request"),
+    CORRECTION_APPLICATION("correctionApplication", "Correction application"),
```
Comment (amoslv1) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/DocumentType.java:14]: new

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/ScannedDocumentType.java [hunk 2]
```diff
@@ -6,14 +6,15 @@
 public enum ScannedDocumentType {
 
     CHERISHED("cherished", "Cherished"),
-    OTHER("other", "Other"),
-    FORM("form", "Form"),
-    COVERSHEET("coversheet", "Coversheet"),
-    REINSTATEMENT_REQUEST("reinstatementRequest", "Reinstatement request"),
     CONFIDENTIALITY_REQUEST("confidentialityRequest", "Confidentiality request"),
-    URGENT_HEARING_REQUEST("urgentHearingRequest", "Urgent hearing request"),
+    CORRECTION_APPLICATION("correctionApplication", "Correction application"),
```
Comment (amoslv1) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/callback/ScannedDocumentType.java:10]: new

**Review by AndreiTrushkov — APPROVED** [review]

**Review by StevensonSi — APPROVED** [review]

File: config/owasp/suppressions.xml [hunk 3]
```diff
@@ -10,4 +10,25 @@
...
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-core@.*$</packageUrl>
+    <cve>CVE-2022-45688</cve>
+  </suppress>
+  <suppress until = "2023-03-15">
+    <notes><![CDATA[
+   file name: json-20220924.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.json/json@.*$</packageUrl>
+    <cve>CVE-2022-45688</cve>
```
Comment (nickhill111) [inline config/owasp/suppressions.xml:25]: this suppression already exists on line 11

File: config/owasp/suppressions.xml [hunk 4]
```diff
@@ -10,4 +10,25 @@
...
+  </suppress>
+  <suppress until = "2023-03-15">
+    <notes><![CDATA[
+   file name: commons-fileupload-1.4.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/commons\-fileupload/commons\-fileupload@.*$</packageUrl>
+    <cve>CVE-2023-24998</cve>
```
Comment (nickhill111) [inline config/owasp/suppressions.xml:32]: same with this one for line 4 (but might be better to delete the one on line 4

**Review by nickhill111 — CHANGES_REQUESTED** [review]

File: config/owasp/suppressions.xml (hunk 4 above)
Comment (amoslv1) [reply config/owasp/suppressions.xml:22]: good spots - removed

File: config/owasp/suppressions.xml (hunk 3 above)
Comment (amoslv1) [reply config/owasp/suppressions.xml]: removed

**Comment by amoslv1** [comment]
Required changes already merged from SSCS-10984

**Pull request closed without merging**